modules/update-upgrading-mcp.adoc

```diff
@@ -0,0 +1,234 @@
+// Module included in the following assemblies:
+//
+// * storage/persistent_storage/persistent-storage-hostpath.adoc
+
+[id="update-upgrading-mcp_{context}"]
+= Delaying the update for specific nodes
+
+There might be situations where you do not want to update specific nodes when you update a cluster. Updating nodes requires the nodes to be cordoned and drained, which marks the nodes as unschedulable and evicts all pods. For example, you can ensure that critical applications stay available during the update process.
```

Comment:
Personally I prefer [the original description](https://github.com/openshift/openshift-docs/pull/34445/files#diff-72d7c105477b4165dbe6af361e4e379589a17c7302264aa164669f6528aab1faR28) about use case. 
```
This helps to make sure the mission critical application stays available during the whole update process even if the upgrade process causes a failure.
It is also useful in scenarios where we can not have a single maintenance window to complete the whole update process.
```
That will help user quickly and clearly understand what use cases this can be applied to.

Reply:
> There might be some scenarios where you want to perform a controlled rollout of an update to specific worker nodes to ensure that mission-critical applications stay available during the update process. After the cluster is updated, you can update those nodes at more appropriate time. This is also useful in scenarios where you want to use more than one maintenance window to complete the update process. Delaying the updates in this manner is referred to as *in-service software updates (ISSU)* or *canary updates*.